src/System.Private.Uri/tests/FunctionalTests/UriBuilderTests.cs

```diff
@@ -13,6 +13,7 @@ public class UriBuilderTests
         //This test tests a case where the Core implementation of UriBuilder differs from Desktop Framework UriBuilder.
         //The Query property will not longer prepend a ? character if the string being assigned is already prepended.
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Difference in behavior")]
```

Comment:
I just added it like that because the test has a comment above explaining the difference in behavior, I can move it to the attribute itself, do you prefer that?